S3 uploader panics

The auth server panics with:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x52d4309]
goroutine 308 [running]:
github.com/gravitational/teleport/lib/events/s3sessions.(*Handler).ListUploads(0xc000916dc0, {0x88c1e10, 0xc000d9c960})
        github.com/gravitational/teleport/lib/events/s3sessions/s3stream.go:202 +0x2c9
github.com/gravitational/teleport/lib/events.(*UploadCompleter).CheckUploads(0xc00224c080, {0x88c1e10?, 0xc000d9c960})
        github.com/gravitational/teleport/lib/events/complete.go:160 +0x77
github.com/gravitational/teleport/lib/events.(*UploadCompleter).Serve(0xc00224c080, {0x88c1e10, 0xc000d9c960})
        github.com/gravitational/teleport/lib/events/complete.go:147 +0x1a7
created by github.com/gravitational/teleport/lib/events.StartNewUploadCompleter
        github.com/gravitational/teleport/lib/events/complete.go:118 +0xdc
```

Observed on v13.1.2, but looks like the bug in the code has been present since the uploader rewrite in 2020: https://github.com/gravitational/teleport/commit/d160507430c7763409a626801f4588656b6c2a19#diff-f928b1b5054b11b47c6447e60675c9b8b06300c0b8f63752705378c3e1ad8596R135

--- a/lib/events/s3sessions/s3stream.go
+++ b/lib/events/s3sessions/s3stream.go
@@ -201,7 +201,7 @@ func (h *Handler) ListUploads(ctx context.Context) ([]events.StreamUpload, error
 				Initiated: *upload.Initiated,
 			})
 		}
-		if !*re.IsTruncated {
+		if !aws.BoolValue(re.IsTruncated) {
 			break
 		}
 		keyMarker = re.KeyMarker

